frontend/src/components/Navbar.tsx

```diff
@@ -0,0 +1,186 @@
+import {
+  Box,
+  Flex,
+  Center,
+  IconButton,
+  Button,
+  Link,
+  Image,
+  Stack,
+  Collapse,
+  useColorModeValue,
+  useBreakpointValue,
+  useDisclosure,
+} from "@chakra-ui/react";
+import React from "react";
+import colors from "../styles/colors";
+import Logo from "../assets/logo.png";
+import { HamburgerIcon, CloseIcon } from "@chakra-ui/icons";
+import { Link as ReactRouterLink } from "react-router-dom";
+
+export default function Navbaor() {
+  const { isOpen, onToggle } = useDisclosure();
+
+  const imageMargin = useBreakpointValue({
+    base: "auto",
+    md: "none",
+  });
+
+  return (
+    <Box>
+      <Flex
+        bg={useColorModeValue("white", "gray.800")}
+        color={useColorModeValue("gray.600", "white")}
+        minH={"60px"}
+        py={{ base: 2 }}
+        px={{ base: 4 }}
+        borderBottom={1}
+        borderStyle={"solid"}
+        borderColor={useColorModeValue("gray.200", "gray.900")}
+        align={"center"}
+      >
+        <Flex
+          flex={{ base: 1, md: "auto" }}
+          display={{ base: "flex", md: "none" }}
+        >
+          <IconButton
+            onClick={onToggle}
+            icon={
+              isOpen ? <CloseIcon w={3} h={3} /> : <HamburgerIcon w={5} h={5} />
+            }
+            variant={"ghost"}
+            aria-label={"Toggle Navigation"}
+          />
+        </Flex>
+        <Flex
+          flex={{ base: 1 }}
+          justify={{ base: "center", md: "start", aligh: "center" }}
+        >
+          <Image
+            minH={"60px"}
+            maxH={"80px"}
+            src={Logo}
+            alt="logo"
+            margin={imageMargin}
+          />
+
+          <Flex display={{ base: "none", md: "flex" }} ml={10}>
+            <DesktopNav />
+          </Flex>
+        </Flex>
+
+        <Stack
+          flex={{ base: 1, md: 0 }}
+          justify={"flex-end"}
+          direction={"row"}
+          spacing={6}
+        >
+          <Button
+            as={"a"}
+            _focus={{ boxShadow: "none" }}
+            fontSize={"sm"}
+            fontWeight={400}
+            variant={"link"}
+            href={"#"}
+          >
+            Sign In
+          </Button>
+          <Button
+            display={{ base: "none", md: "inline-flex" }}
+            fontSize={"sm"}
+            fontWeight={600}
+            color={"white"}
+            bg={colors.primary}
+            _hover={{
+              bg: colors.secondary,
+            }}
+          >
+            Sign Up
+          </Button>
+        </Stack>
```

Comment:
Is the sign up button supposed to disappear on mobile?

Reply:
Yeah, otherwise it might be too crowded. The sign up button is most likely going to be commented out sometime in a future iteration but for the time being we can leave it in and discuss with Chaisi where it would best be placed.